src/main/java/meditracker/argument/Argument.java

```diff
@@ -24,13 +25,20 @@ public abstract class Argument {
      * @param prompt Guided prompt message
      * @param help Help message for additional context
      * @param isOptional Whether this argument is optional or required
+     * @param hasNoValue Whether this argument requires a value specified with it
```

Comment:
The variable name hasNoValue does not match the docs, maybe it could be changed to needsValue. Try to avoid double negatives as it may cause confusion.

Reply:
Resolved in a851c66